app/views/selections/_newStation.html.erb

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
Remove extra blank lines. 
Indentation should always be exactly two spaces.
File names should be underscores (snake case?), not camel-case.